fix: add note preventing user from changing nameserver (#15453)
github/docs: data/reusables/enterprise_installation/preventing-nameservers-change.md

	{% note %}

> 	**Note**: The 127.0.0.1 entry is **required** to be the first entry in the list here. Removing it or having another entry as the first entry will break GitHub Enterprise. It's also not supported to add options that affect the ordering here such as "options rotate"

Reviewer: ```suggestion
	**Note**: The 127.0.0.1 entry is required to be the first entry in the list. Do not remove the 127.0.0.1 entry, add another entry above the 127.0.0.1 entry, or use options that affect the ordering of entries, such as "options rotate".
```
